server/src/test/java/org/elasticsearch/index/mapper/KeywordFieldMapperTests.java

```diff
@@ -772,13 +772,12 @@ public void testDocValuesLoadedFromStoredSynthetic() throws IOException {
         assertScriptDocValues(mapper, "foo", equalTo(List.of("foo")));
     }
 
-    public void testFieldTypeWithSkipDocValues_LogsDbMode() throws IOException {
-        assumeTrue("Needs feature flag to be enabled", FieldMapper.DOC_VALUES_SPARSE_INDEX.isEnabled());
-
+    public void testFieldTypeWithSkipDocValues_LogsDbModeDisabledSetting() throws IOException {
```

Comment:
Now all these tests written in such a way that they work both for snapshot and release builds.